src/dispatch/models.py

```diff
@@ -140,11 +141,20 @@ class ResourceBase(DispatchBase):
 
 
 class ContactBase(DispatchBase):
-    email: EmailStr
+    email: str
```

Comment:
@kevgliss figured u should see this change specifically

Reply:
I did test this E2E by modifying an email to some junk value and it works as expected. I get the 422 before, and valid request + default value after this change.

Reply:
Hmmm, this seems a bit counterintuitive to me? If I put in a bad email, I don't put in a valid email I get a 200 and the email is "unknown@email.com"?

Reply:
agreed this is probably not ideal, it begs the question how we ended up with one in the first place. it might be hard to figure out at this point. 

without knowing for sure how it ended up in the database, does it make sense to have a dedicated view and model specifically for read operations, to have the robustness in the event that one somehow slips by (otherwise the entire component stops working, when we really just want the bad value to get handled in some way).